test/functional/test_framework/util.py

```diff
@@ -34,13 +34,13 @@ def assert_approx(v, vexp, vspan=0.00001):
         raise AssertionError("%s > [%s..%s]" % (str(v), str(vexp - vspan), str(vexp + vspan)))
 
 
-def assert_fee_amount(fee, tx_size, fee_per_kB):
-    """Assert the fee was in range"""
-    target_fee = round(tx_size * fee_per_kB / 1000, 8)
+def assert_fee_amount(fee, tx_size, feerate_per_kvB):
```

Comment:
Wondering if `feerate_per_kvB` is better than `fee_per_kvB`. Technically `_per_kvB` indicates a rate, so `feerate_per_kvB` may sound like `rate of a rate`. 

No hard opinion in either way.

Reply:
Good point, updated to the more precise `feerate_BTC_kvB`. Let me know if that's better.

Reply:
Sorry I am probably bikeshedding here, this one seems even more confusing. To me the original `fee_per_kvB` seems reasonable.

Reply:
Yes, if you run something like `git grep -ni "btc_kvb\|sat_vb"` from repo root, you'll see that we have been working to upgrade and standardize how we describe fee rates

- from fee to fee rate or feerate
- from `fee per (denominator without specifying the numerator)` to `fee rate in numerator/denominator` 
- from fee rate units like BTC or satoshis or kB or B, to BTC/kvB and sat/vB